gensim/models/keyedvectors.py

```diff
@@ -2291,7 +2291,7 @@ def adjust_vectors(self):
             ngram_hashes = ft_ngram_hashes(w, self.min_n, self.max_n, self.bucket, self.compatible_hash)
             for nh in ngram_hashes:
                 word_vec += self.vectors_ngrams[nh]
-            word_vec /= len(ngram_hashes) + 1
```

Comment:
This "fix" is plainly incorrect: the value of `word_vec` is composed from 1 full-word vector (from `vectors_vocab`), plus `len(ngram_hashes)` ngram vectors. So to get the average of `len(ngram_hashes) + 1` separate vectors, you divide by `len(ngram_hashes) + 1`.